VictoriaAlbert/VictoriaAlbert/DetailViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DetailViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Madushani Lekam Wasam Liyanage on 11/10/16.
+//  Copyright © 2016 Madushani Lekam Wasam Liyanage. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+    
+    
+    @IBOutlet weak var museumObjectImage: UIImageView!
+    @IBOutlet weak var detailLabel: UILabel!
+    
+    
+    var object: MuseumObject?
+    var apiEndPoint: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = object?.objectName
+        
```

Comment:
There should only ever be one blank line or two at most. Whichever you choose, be consistent.